release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -514,9 +514,46 @@ For more information, see xref:../networking/metallb/about-metallb.adoc#about-me
 [id="ocp-4-10-olm"]
 === Operator lifecycle
 
+[id="ocp-4-10-copied-csvs"]
+==== Disabling copied CSVs to support large clusters
+
+When an Operator is installed by Operator Lifecycle Manager (OLM), a simplified copy of its cluster service version (CSV) is created in every namespace that the Operator is configured to watch. These CSVs are known as copied CSVs and communicate to users which controllers are actively reconciling resource events in a given namespace.
```

Comment:
Review ISG on "that" vs. "which" for restrictive clauses here. I'm not sure if that's even applicable here, as the phrasing of the final sentence sends my tired brain on a loop. :) 

Is it: "These CSVs are known as copied CSVs. They identify controllers that are actively reconciling resource events in a given namespace." ?

Reply:
The latter, yeah.